.chglog/consolidate-changelogs.js

```diff
@@ -1,13 +1,18 @@
-const readline = require("readline");
-const path = require("path");
-const fs = require("fs");
+import readline from "node:readline";
+import path from "node:path";
+import fs from "node:fs";
+import {pathToFileURL} from "node:url";
+
+// Using CommonsJS require.resolve as long as import.meta.resolve is experimental
+import {createRequire} from "node:module";
+const require = createRequire(import.meta.url);
 
 function handleDependencyBump(line) {
 	line = line.replace("[@ui5](https://github.com/ui5)", "@ui5");
 	const moduleMatch = line.match(/Bump (@ui5\/[^\s]+).*to ([^ ]+)/);
 	if (moduleMatch) {
 		const [, moduleName, moduleVersion] = moduleMatch;
-		const moduleDir = path.dirname(require.resolve(moduleName));
+		const moduleDir = path.dirname(pathToFileURL(require.resolve(moduleName)));
```

Comment:
Did you test the script?
My gut feeling is that this won't work for packages without a default export entry. We might need to use `${moduleName}/package.json` instead. At lest that's what I also did in https://github.com/SAP/ui5-tooling/blob/3b1338bf53513e9ea9ecd9a969bd624a927af3df/scripts/buildSchema.js

Reply:
Adjusted.